<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>49</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Informace o stahování</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>25.6 / 120.7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Zadejte URL videa</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Název videa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Délka videa:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>&lt;Vyberte formát&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mp3 (pouze zvuk)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mp4 (video)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>&lt;Vyberte kvalitu&gt;</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>35</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Najít</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Zrušit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actiony2mate_com"/>
    <addaction name="actionzdrojovy_kod"/>
    <addaction name="separator"/>
    <addaction name="actionNastaven"/>
   </widget>
   <widget class="QMenu" name="menuHistorie">
    <property name="title">
     <string>Historie</string>
    </property>
    <widget class="QMenu" name="menu1">
     <property name="title">
      <string>1.</string>
     </property>
     <addaction name="actionPou_t"/>
     <addaction name="actionOtev_t"/>
     <addaction name="separator"/>
     <addaction name="actionD_lka"/>
    </widget>
    <widget class="QMenu" name="menu2">
     <property name="title">
      <string>2.</string>
     </property>
     <addaction name="actionPou_t_2"/>
     <addaction name="actionOtev_t_2"/>
     <addaction name="separator"/>
     <addaction name="actionD_lka_2"/>
    </widget>
    <widget class="QMenu" name="menu3">
     <property name="title">
      <string>3.</string>
     </property>
     <addaction name="actionPou_t_3"/>
     <addaction name="actionOtev_t_3"/>
     <addaction name="separator"/>
     <addaction name="actionD_lka_3"/>
    </widget>
    <widget class="QMenu" name="menu4">
     <property name="title">
      <string>4.</string>
     </property>
     <addaction name="actionPou_t_4"/>
     <addaction name="actionOtev_t_4"/>
     <addaction name="separator"/>
     <addaction name="actionD_lka_4"/>
    </widget>
    <widget class="QMenu" name="menu5">
     <property name="title">
      <string>5.</string>
     </property>
     <addaction name="actionPou_t_5"/>
     <addaction name="actionOtev_t_5"/>
     <addaction name="separator"/>
     <addaction name="actionD_lka_5"/>
    </widget>
    <addaction name="menu1"/>
    <addaction name="menu2"/>
    <addaction name="menu3"/>
    <addaction name="menu4"/>
    <addaction name="menu5"/>
    <addaction name="separator"/>
    <addaction name="actionSmazat_historii"/>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menuHistorie"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiony2mate_com">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/y2mate.ico</normaloff>:/images/y2mate.ico</iconset>
   </property>
   <property name="text">
    <string>y2mate.com</string>
   </property>
  </action>
  <action name="actionzdrojovy_kod">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/github.ico</normaloff>:/images/github.ico</iconset>
   </property>
   <property name="text">
    <string>zdrojový kód</string>
   </property>
  </action>
  <action name="actionNastaven">
   <property name="text">
    <string>Nastavení</string>
   </property>
  </action>
  <action name="actionSmazat_historii">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/delete.ico</normaloff>:/images/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Smazat historii</string>
   </property>
  </action>
  <action name="actionPou_t">
   <property name="text">
    <string>Použít</string>
   </property>
  </action>
  <action name="actionPou_t_2">
   <property name="text">
    <string>Použít</string>
   </property>
  </action>
  <action name="actionPou_t_3">
   <property name="text">
    <string>Použít</string>
   </property>
  </action>
  <action name="actionPou_t_4">
   <property name="text">
    <string>Použít</string>
   </property>
  </action>
  <action name="actionPou_t_5">
   <property name="text">
    <string>Použít</string>
   </property>
  </action>
  <action name="action_nazev">
   <property name="text">
    <string>[nazev]</string>
   </property>
  </action>
  <action name="action_delka">
   <property name="text">
    <string>[delka]</string>
   </property>
  </action>
  <action name="action_nazev_2">
   <property name="text">
    <string>[nazev]</string>
   </property>
  </action>
  <action name="action_delka_2">
   <property name="text">
    <string>[delka]</string>
   </property>
  </action>
  <action name="action_nazev_3">
   <property name="text">
    <string>[nazev]</string>
   </property>
  </action>
  <action name="action_delka_3">
   <property name="text">
    <string>[delka]</string>
   </property>
  </action>
  <action name="action_nazev_4">
   <property name="text">
    <string>[nazev]</string>
   </property>
  </action>
  <action name="action_delka_4">
   <property name="text">
    <string>[delka]</string>
   </property>
  </action>
  <action name="action_nazev_5">
   <property name="text">
    <string>[nazev]</string>
   </property>
  </action>
  <action name="action_delka_5">
   <property name="text">
    <string>[delka]</string>
   </property>
  </action>
  <action name="actionOtev_t">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="actionOtev_t_2">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="actionOtev_t_3">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="actionOtev_t_4">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="actionOtev_t_5">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="actionD_lka">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Délka:</string>
   </property>
  </action>
  <action name="actionD_lka_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Délka:</string>
   </property>
  </action>
  <action name="actionD_lka_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Délka:</string>
   </property>
  </action>
  <action name="actionD_lka_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Délka:</string>
   </property>
  </action>
  <action name="actionD_lka_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Délka:</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
